cms/models/pagemodel.py

```diff
@@ -674,7 +675,8 @@ def publish(self, language):
 
         if not self.pk:
             self.save()
-            # be sure we have the newest data including mptt
+
+        # be sure we have the newest data including mptt
```

Comment:
Minor thing, feel free to ignore, I'll rephrapse as "be sure we have the newest data including tree infomation)"